project-ideas/fulfillment-center-mgmt/readme.md

```diff
@@ -7,44 +7,64 @@
 
 
 Order fulfillment is 3 step process,
-*   Step 1:  * The staff gets the list of [Outstanding orders](https://github.com/hotwax/oms-documentation/blob/user-guides-pub/documents/integrate-with-hotwax/api/fulfillment/apis/outstanding-orders.md). 
+
+### Step 1: 
+The staff gets the list of [Outstanding orders](https://github.com/hotwax/oms-documentation/blob/user-guides-pub/documents/integrate-with-hotwax/api/fulfillment/apis/outstanding-orders.md). 
+
   * The user then starts the fulfillment process for set of orders by creating a [Fulfillment wave of orders](createOrderFulfillmentWave.md). A [PickList](PickList.md) is returned for the user to go pick items for preparing the shipments. On success of createOrderFulfillmentWave the OrderItems are tagged "isPicked: Y" in SOLR doc. 
-      *  Background process: [Shipments](createShipment.md) are created for orders. 
-        *  Based on the inventory allocation rules inventory from a certain location is assigned to [Shipment](/udm/intermediate/Shipment.md), 
-        *  A background process is initiated to get shipping labels from the shipping provider.
+     *  Background process: [Shipments](createShipment.md) are created for orders. 
+
+        - Based on the inventory allocation rules inventory from a certain location is assigned to [Shipment](/udm/intermediate/Shipment.md). **Where are these inventory allocation rules located in the OMS?**
```

Comment:
I agree, 
The Inventory Reservation is created by the Order Brokering process. 
The scope of Fulfillment center is limited to `fulfill the order` or `reject it`. 
It is possible that during order processing at fulfillment center, the systemic inventory is zero (or -ve) and the order is packed and shipped.